content/contribute/add_phone_to_plugin.md

```diff
@@ -0,0 +1,234 @@
+
+2. Find the `COMMON_FILES` list and add the correct entry for the `y0000000000xx.cfg` file.
+
+    Example:
+
+    ```python
+    COMMON_FILES = [
+        # (...)
+        {'y000000000066.cfg', u'T46S(T48S,T42S,T41S)-66.85.0.5.rom', 'model.tpl'},
+        # (...)
+    ]
+    ```
+
+If the phone is a DECT model (not the case for the T41S, but we will ignore this fact), you must
+add the model to the `COMMON_FILES_DECT` list instead.
+
+Example for the W60B:
+
+```python
+COMMON_FILES_DECT = [
+    # (...)
+    {
+        'filename': u'y000000000077.cfg',
+        'fw_filename': u'W60B-77.85.0.20.rom',
+        'handsets_fw': {
+            'w53h': u'W53H-88.85.0.20.rom',
+            'w56h': u'W56H-61.85.0.20.rom',
+            'w59r': u'W59R-115.85.0.20.rom',
+            'cp930w': u'CP930W-87.85.0.20.rom',
+        },
+        'tpl_filename': u'dect_model.tpl',
+    },
+]
+```
+
+### `plugins/xivo-yealink-v85/pkgs/pkgs.db`
+
+This file is where the firmware download information is located.
+
+We need to add a *package definition* and a *file definition*.
+
+The *package definition* is usually at the top of the `pkgs.db` file:
+
+```
+[pkg_T46S_T48S_T42S_T41S-fw]
+description: Firmware for Yealink T46S T48S T42S and T41S
+description_fr: Micrologiciel pour Yealink T46S T48S T42S et T41S
+version: 66.85.0.5
+files: T46S_T48S_T42S_T41S-fw
+install: yealink-fw
+```
+
+The `files` option naming is important: it will use the files from the file section with the
+corresponding name, prepended by `file_`. So `file_T46S_T48S_T42S_T41S-fw`.
+
+Also, the package will use the `yealink-fw` install method. It is already in the file, so **you do
+not need to add it**, but to be clear this is its content:
+
+```
+[install_yealink-fw]
+a-b: cp *.rom firmware/
+```
+
+As you can see, it will copy the ROM file to the `firmware` directory. The `firmware` directory
+in question is located in the `tftpboot` directory in the installed plugin, located at
+`/var/lib/wazo-provd/plugins/wazo-yealink-v85/var/tftpboot/firmware`. It is possible to have more
+than one step in the install method. To see how it is done, look at what is done with the language
+packages already present in the `pkgs.db` file.
+
+The *file definition* is usually at the bottom of the `pkgs.db` file:
+
+```
+[file_T46S_T48S_T42S_T41S-fw]
+filename: T46S(T48S,T42S,T41S)-66.85.0.5.rom
+url: http://support.yealink.com/forward2download?path=ZIjHOJbWuW/DFrGTLnGypkTJASVqwzX/x71yaViKhXSKOgYSKl0Pb7N9IXFpYf7YAOhOJmV0fGjbcYgmf0b9uxGdQ1IFTWkfgTz2m4g5Uoy1NWm3/S6JBTsplusSymbolM5UOVemCq9Kf2pvmyoc=
+size: 25559152
+sha1sum: 091d469ba3ce13b424a5212fb74f0e1b59c31a6a
+```
+
+The size is in bytes and the sha1sum is obtained using the `sha1sum` command on a GNU/Linux system.
+
+### `plugins/xivo-yealink-v85/templates/T41S.tpl`
+
+It is necessary to create a `T41S.tpl` file or copy-paste an existing file and rename it to
+`T41S.tpl`.Make sure that its content is correct.
+
+In our case it should be:
+
+```jinja
```

Comment:
I think you should add a link to the Jinja template developer doc https://jinja.palletsprojects.com/en/2.11.x/templates/